<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="IQTLAPTOP-ThejaswiniBhuma" failures="0" tests="2" name="com.quantela.codenet.qa.qpf_wallet_mob_ui_test.utils.LoginTest" time="1.018" errors="1" timestamp="2023-11-05T16:36:24 IST" skipped="1">
  <testcase classname="com.quantela.codenet.qa.qpf_wallet_mob_ui_test.utils.LoginTest" name="citizenUserSignin" time="0.001">
    <skipped/>
  </testcase> <!-- citizenUserSignin -->
  <testcase classname="com.quantela.codenet.qa.qpf_wallet_mob_ui_test.utils.LoginTest" name="suiteSetup" time="1.017">
    <error message="Cannot invoke &quot;io.appium.java_client.android.AndroidDriver.manage()&quot; because &quot;com.quantela.codenet.qa.qpf_wallet_mob_ui_test.core.BaseTest.driver&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "io.appium.java_client.android.AndroidDriver.manage()" because "com.quantela.codenet.qa.qpf_wallet_mob_ui_test.core.BaseTest.driver" is null
at com.quantela.codenet.qa.qpf_wallet_mob_ui_test.core.BaseTest.AndroidDriverConfig(BaseTest.java:67)
at com.quantela.codenet.qa.qpf_wallet_mob_ui_test.core.BaseTest.suiteSetup(BaseTest.java:41)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:382)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- suiteSetup -->
</testsuite> <!-- com.quantela.codenet.qa.qpf_wallet_mob_ui_test.utils.LoginTest -->
